### ISSUE TITLE
Unable to login to smarteefi

### ISSUE THREAD

**Issue by snack0verflow** [prasad-edlabadka/homebridge-smarteefi#4]
```
[23/10/2023, 21:54:59] [Smarteefi] Logging in to the server https://www.smarteefi.com/site...
[23/10/2023, 21:55:00] [Smarteefi] Unable to login. Please verify user id and password and restart homebridge.
[23/10/2023, 21:55:00] [Smarteefi] Unable to login. Retrying after 60 seconds...
```
The user and pass are correct and homebridge has been restarted, I am able to login to https://smarteefi.web.app/start/tabs/dashboard via browser.
Did they break API? Or is the login URL changed?

**Comment by prasad-edlabadka** [comment]
Yes. They have changed URL. Will work to update this.

**Comment by prasad-edlabadka** [comment]
Released v0.1.0 with fixes. Please test and let me know.

**Comment by prasad-edlabadka** [comment]
Updated plugin to 0.1.2. Closing for now.
